docs/useful_links.md

```diff
@@ -1,11 +1,12 @@
+Find below a list of links that show the capabilities of our Gen3 software stack tested by and further developed by users and initiatives in the research community all over the globe. These are fantastic resources to explore use cases of Gen3 and may be of help to new and experienced users/operators alike of a Gen3 Data Commons.
+
 > 🟢 Note: We emphasize that we are not responsible for the content and opinions on the third-party webpages listed below.
 
 1. Working with on premises data and servers:
 The gen3 system is optimized to deploy on cloud systems and work with cloud buckets. The Oregon Health & Science University (OHSU) has developed [a collection of extensions](https://github.com/ohsu-comp-bio/compose-services/tree/onprem) to enable gen3 to work in a non aws environment.  Read this [overview](https://github.com/ohsu-comp-bio/compose-services/blob/onprem/onprem/README.md) for more information.
 2. A group of users shared their experiences with setting up their Gen3 Data Commons on a local desktop using Compose Services in August 2020 in form of three videos: [Gen3 Data Commons Setup Part 1](https://www.youtube.com/watch?v=xM54O4aMpWY), [Gen3 Data Commons Setup Part 2](https://www.youtube.com/watch?v=iMmCxnbHpGo), and [Data Upload](https://www.youtube.com/watch?v=F2EOtHPg6g8&feature=youtu.be). Please note, that the content in these videos might not reflect the current status of the Compose-Services repository. Referring to the video part 1, the following is outdated: the format of the `user.yaml` reflects the one shown in the Fence repository and the arborist DB setup is up to date.
 3. A stand-alone data dictionary viewer for schema.json artifacts was published [here](https://github.com/bioteam/dictionary-visualizer).
-4. A second group of users shared experiences with setting up Compose-Services and wrote documentation for different use cases (admin, user) as well as details on its deployment [here](https://github.com/umccr/gen3-doc). This group also created a [client](https://github.com/umccr/g3po) to work with Gen3. 
+4. The [Australian BioCommons group](https://www.biocommons.org.au/gen3-project) has implemented the Gen3 software stack for easier management and sharing of human genome data in Australia. Browse through the [detailed documentation](https://github.com/umccr/gen3-doc) of how they setup and deployed Gen3 including a [customized Data Dictionary](https://github.com/umccr/umccr-dictionary) and guides for users and admins on working with the [production environment](https://github.com/umccr/gen3-doc/tree/main/cloud). This group also wrote a [client "g3po"](https://github.com/umccr/g3po) to interact with Gen3 APIs and tested Gen3's capability to set [granular access to data files using authz and GA4GH Passport Visa consent codes](https://github.com/umccr/gen3-doc/tree/main/submit/agha-gdr-demo) in combination with CILogon as [a new authentication method](https://github.com/uc-cdis/fence/pull/896). 
```

Comment:
i'm not sure about the last part `in combination with CILogon as a new authentication method` since it links to our own codebase - now that their PR was merged, it's just part of Gen3, not a community tool. but if you want to give credit here that's fine

Reply:
yes I wanted to give credit exactement. :)